README.md

```diff
@@ -203,6 +203,8 @@ Returns HTTP 404 if no order with specified orderHash was found.
 
 ### GET /v0/order_book
```

Comment:
were we going to rename this to "orderbook" here as well?

Reply:
I'm going to modify the channel name to `orderbook` as well, since it is much more similar to that HTTP endpoint